tests/test_helpers.py

```diff
@@ -975,3 +977,51 @@ def test_populate_with_cookies():
 )
 def test_parse_http_date(value, expected):
     assert parse_http_date(value) == expected
+
+
+def test_netrc_from_env():
```

Comment:
Same here — use built-in fixtures plz.

Reply:
Yep, done. I didn't know pytest shipped with somehing for tmp_path!

I'll do the same for the mock.